using-mobile-extensions/adobe-target/README.md

```diff
@@ -505,6 +505,7 @@ If you need to update SDK configuration, programmatically, use the following inf
 | target.timeout | Time, in seconds, to wait for a response from Target servers before timing out. |
 | target.environmentId | Environment ID you want to use, if this is left blank, the default production environment will be used. |
 | target.propertyToken | `at_property token` value, which is generated from the Target UI. If this value is left blank, no token is sent in the Target network calls. |
+| target.previewEnabled | Boolean parameter which can be used to enable/disable Target Preview. If not specified, then Preview will be enabled by default. |
```

Comment:
Please add a comma between "parameter" and "which".

Reply:
done